src/Microsoft.Azure.WebJobs.Extensions.ServiceBus/ServiceBusTriggerAttribute.cs

```diff
@@ -83,6 +83,16 @@ public string SubscriptionName
             get { return _subscriptionName; }
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the sessions are enabled.
+        /// </summary>
+        public bool IsSessionsEnabled { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Session Id
+        /// </summary>
+        public string SessionId { get; set; }
```

Comment:
Does this ever get used?

Reply:
I think this is a remnant from a previous design. I think we decided not to expose SessionId via attribute